docs/source/commandline/commandline-all.rst

```diff
@@ -648,125 +733,238 @@ Core State/Playback Options
+    Hard-codes the size for snapshots and movie recording.  By default, MAME
+    will create snapshots at the system's current resolution in raw pixels, and
+    will create movies at the system's starting resolution in raw pixels.  If
+    you specify this option, then MAME will create both snapshots and movies at
+    the size specified, and will bilinear filter the result.  Note that this
+    size does not automatically rotate if the system is vertically oriented.
+
+    The default is ``auto``.
 
 .. _mame-commandline-snapview:
 
 **-snapview** *<viewname>*
 
-	Specifies the view to use when rendering snapshots and movies. By default, both use a special 'internal' view, which renders a separate snapshot per screen or renders movies only of the first screen. By specifying this option, you can override this default behavior and select a single view that will apply to all snapshots and movies. Note that <viewname> does not need to be a perfect match; rather, it will select the first view whose name matches all the characters specified by <viewname>.
+    Specifies the view to use when rendering snapshots and movies.
 
-	For example, **-snapview native** will match the "Native (15:14)" view even though it is not a perfect match. <viewname> can also be 'auto', which selects the first view with all screens present. The default value is ``internal``.
+    By default, both use a special 'internal' view, which renders a separate
+    snapshot per screen or renders movies only of the first screen.  By
+    specifying this option, you can override this default behavior and select a
+    single view that will apply to all snapshots and movies.  Note that
+    <viewname> does not need to be a perfect match; rather, it will select the
+    first view whose name matches all the characters specified by <viewname>.
+
+    For example, **-snapview native** will match the "Native (15:14)" view even
+    though it is not a perfect match.  <viewname> can also be 'auto', which
+    selects the first view with all screens present.
+
+    The default value is ``internal``.
 
 .. _mame-commandline-nosnapbilinear:
 
 **-[no]snapbilinear**
 
-	Specify if the snapshot or movie should have bilinear filtering	applied. Shutting this off can make a difference in some performance while recording video to a file. The default is ON (**-snapbilinear**).
+    Specify if the snapshot or movie should have bilinear filtering	applied.
+    Shutting this off can make a difference in some performance while recording
+    video to a file.
```

Comment:
The main motivation for this is exact colours and precise edges vs uneven pixels, not performance.

Reply:
I'm going to put this one down as "Probably doesn't need explaining at all."
I'd had some misgivings on the bilinear performance thing-- that'd be true 20 years ago, but not so much now. I'll put forward my suggested change and we can talk from there on this one.